tests/datatypes/test_list.py

```diff
@@ -4,6 +4,18 @@
 
 
 class ListTests(TranspileTestCase):
+
+    def assertOrdering(self, col1, col2):
+        """
+        runs assertCodeExecution with col1 [<, <=, >, >=] col2
+        :param col1: array like
+        :param col2: array like
+        """
+        set_up = "print('>>> x = {col1}')\nprint('>>> y = {col2}')\n".format(col1=col1, col2=col2)
+        operators = ['>', '>=', '<', '<=']
+        comparisons = ["\nprint('>>> x {o} y')\nprint({col1} {o} {col2})\n".format(col1=col1, o=o, col2=col2) for o in operators]
```

Comment:
The only comment here is about formatting. For readability, long list comprehensions can be broken into multiple lines. You can also use the `adjust()` method in the test utilities to make the code snippet easier to read:
```
    comparisons = [
         adjust("
             print('>>> x {o} y')
             print({col1} {o} {col2})
        ").format(col1=col1, o=o, col2=col2) 
        for o in ['>', '>=', '<', '<=']
    ]
```

`adjust()` strips all left-hand-side whitespace, so you can have code snippets in multiline comments, rather than using \n in single line strings.